server/notification-providers/call-me-bot.js

```diff
@@ -0,0 +1,23 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class CallMeBot extends NotificationProvider {
+    name = "CallMeBot";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        try {
+            const url = new URL(notification.callMeBotEndpoint);
+            url.searchParams.delete("text");
+            await axios.get(`${url}&text=${encodeURIComponent(msg)}`);
```

Comment:
You could simply do the folllowing which does not have implicite assumptions about the url-parameters (f.ex. containing a `?` previously)

I have not tested if `toString` is actually nessesary.

```suggestion
            url.searchParams.set("text", msg);
            await axios.get(url.toString());
```

Reply:
I tested it and it's working as expected. The `toString` is required to parse the url correctly.